packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ReactElement, ReactNode, useId } from "react";
+import styles from "./SelectList.module.css";
+import classNames from "classnames";
+import Typography from "../Typography/Typography";
+import SelectOption from "./SelectOption";
+
+const iconSize = {
+  sm: 20,
+  md: 24,
+  lg: 24,
+} as const;
+
+const SelectList = ({
+  children,
+  disabled = false,
+  error = false,
+  errorText,
+  helperText,
+  label,
+  onChange,
+  placeholderText,
+  selectedValue = "",
+  size = "md",
+}: {
+  /**
+   * One or more SelectList.Option components.
+   */
+  children: ReactNode;
+  /**
+   * true if the select dropdown is disabled
+   */
+  disabled?: boolean;
+  /**
+   * true if there is an error with the selection
+   */
+  error?: boolean;
+  /**
+   * Text shown below select box if there is an input error.
+   * Should only have a value if error = true
+   */
+  errorText?: string;
+  /**
+   * Text shown below select box
+   */
+  helperText?: string;
+  /**
+   * Text shown above select box
+   */
+  label?: string;
+  /**
+   * The callback to be called when an option is selected
+   */
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  /**
+   * Text showing in select box if no option has been chosen
+   *
+   */
+  placeholderText: string;
```

Comment:
@nathan-ong should this be optional?

Reply:
Or, are you saying if there's no placeholder, we default to showing option 1?